lib/strategies/android-chrome.js

```diff
@@ -9,6 +9,30 @@ function AndroidChromeStrategy(options) {
 AndroidChromeStrategy.prototype = Object.create(BaseStrategy.prototype);
 AndroidChromeStrategy.prototype.constructor = AndroidChromeStrategy;
 
+BaseStrategy.prototype._attachListeners = function () {
+  var self = this;
+
+  self.inputElement.addEventListener('keydown', function (event) {
+    if (keyCannotMutateValue(event)) { return; }
+    self._unformatInput(event);
+  });
+  self.inputElement.addEventListener('keypress', function (event) {
+    if (keyCannotMutateValue(event)) { return; }
+    self._unformatInput(event);
+  });
+  self.inputElement.addEventListener('keyup', function (event) {
+    self._reformatInput(event);
+  });
+  self.inputElement.addEventListener('input', function (event) {
+    // Safari AutoFill fires CustomEvents -- mark the input as unformatted without actually unformatting the current value
```

Comment:
I understand that for our purposes, we only care about hosted fields, but this library can be used outside of hosted fields and I'd be interested to know about cc autofill behavior in a normal form.